includes/cli/class-sensei-db-validate-progress-command.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Sensei_DB_Validate_Progress_Command class file.
+ *
+ * @package sensei
+ */
+
+use Sensei\Internal\Migration\Validations\Progress_Validation;
+
+defined( 'ABSPATH' ) || exit;
```

Comment:
JS-style :)

Reply:
Haha, I've copied it from another file... Fixed in 57e53d55c776563f288427f5f82713a541709504. Btw, this actually is a valid logic. 🙂